webapp/components/test-results-history-grid.html

```diff
@@ -97,35 +96,34 @@
         };
       }
 
-      async onPathChanged(path) {
+      // eslint-disable-next-line no-unused-vars
+      loadResults(path, query) {
```

Comment:
Perhaps this needs to be `async` too for consistency? Or at least the early return when `!path` should be a thenable?